src/test/java/core/basesyntax/service/TransactionParserServiceTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.TransactionParserServiceImpl;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TransactionParserServiceTest {
+    private static final String INCORRECT_FIRSLINE_PATH =
+            "src/test/resources/falseInput.csv";
+    private static final String DEFAULT_DATA =
+            "src/test/resources/input.csv";
+    private static ReaderService readerService;
+    private static TransactionParserService transactionParserService;
+    private static List<FruitTransaction> defaultData;
+
+    @BeforeClass
+    public static void beforeAll() {
+        readerService = new ReaderServiceImpl();
+        transactionParserService = new TransactionParserServiceImpl();
+        defaultData = List.of(new FruitTransaction("b", "banana", 20),
```

Comment:
move to the test since it is used only in a single test and name the vars there `expected` and `actual`